pkg/controller/kibana/pod/pod.go

```diff
@@ -56,10 +57,10 @@ func readinessProbe(useTLS bool) corev1.Probe {
 		SuccessThreshold:    1,
 		TimeoutSeconds:      5,
 		Handler: corev1.Handler{
-			HTTPGet: &corev1.HTTPGetAction{
-				Port:   intstr.FromInt(HTTPPort),
-				Path:   "/login",
-				Scheme: scheme,
+			Exec: &corev1.ExecAction{
+				Command: []string{"bash", "-c",
+					fmt.Sprintf(`curl -o /dev/null -w "%%{http_code}" %s://127.0.0.1:%d/login -k -s`, scheme, HTTPPort),
```

Comment:
Why is there 2 `%` here vs. only one in the apm case?

Reply:
`%%` is for `%` in a `fmt.Sprintf` template.

```sh
# fmt.Sprintf
pkg/controller/kibana/pod/pod.go: curl -o /dev/null -w "%%{http_code}" ...
pkg/controller/apmserver/pod.go:  curl -o /dev/null -w "%%{http_code}" ...
```

```sh
# string
pkg/controller/kibana/driver_test.go: curl -o /dev/null -w "%{http_code}" ...
pkg/controller/kibana/driver_test.go: curl -o /dev/null -w "%{http_code}" ...
pkg/controller/apmserver/apmserver_controller_test.go: curl -o /dev/null -w "%{http_code}" ...
```

Reply:
👍 I misread the code: there is the same for APM (I was looking at the test)